code/game/objects/effects/spawners/lootsite.dm

```diff
@@ -3,7 +3,8 @@
 	icon = 'icons/effects/landmarks_static.dmi'
 	icon_state = "loot_site"
 	///This is the loot table for the spawner. Try to make sure the weights add up to 1000, so it is easy to understand.
```

Comment:
this is of course not your fault but rather just something i'm noticing: to whomever set this standard, you're a pretty fucked up guy